clean_project/clean_file.py

```diff
@@ -0,0 +1,21 @@
+import unicodedata
+import sys
+
+table = dict.fromkeys(i for i in xrange(sys.maxunicode)
+                      if unicodedata.category(unichr(i)).startswith('P'))
+
+
+def remove_punctuations(line):
+    a = line.decode('utf-8').translate(table)
```

Comment:
please inline this variable (return it directly, as you don't make any obvious use of `a`)